src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,18 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = [];
+
+const booksSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    remove(store, action) {
+      return store.filter((book) => book.id !== action.payload);
+    },
+    add(store, action) {
+      store.push(action.payload);
+    },
+  },
+});
+
+export default booksSlice.reducer;
```

Comment:
- [ ] You have not exported the `add` and `remove` action like so 👇🏾 

```javascript
export const { add, remove } = booksSlice.actions
```

For better readability your reducers should be called `addBook` and `removeBook`